specification/storage/data-plane/Microsoft.FileStorage/stable/2025-01-05/file.json

```diff
@@ -630,12 +636,12 @@
                 "format": "date-time-rfc1123",
                 "description": "A UTC date/time value generated by the service that indicates the time at which the response was initiated."
               },
-              "x-ms-share-usage-bytes": {
+              "x-ms-file-share-usage-bytes": {
                 "type": "integer",
                 "format": "int64",
                 "description": "The \"live share\" portion of the data that the customer will be billed for in the soft-deleted capacity (logical storage size)."
```

Comment:
Description can be changed to - "Returned only for provisioned v2 file shares. Returns an approximate used storage size of the share, in bytes."

Reply:
Fixed.